optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties

```diff
@@ -17,7 +16,6 @@ HomeProducer.ProjectAuthoring=Project Authoring
 HomeProducer.ArtifactRepository=Artifact Repository
 HomeProducer.Administration=Administration
 HomeProducer.DeployYourArtifacts=Deploy your Artifacts
-
```

Comment:
xml files look good too .
But properties files loose empty lines (that's strange because we explicitely configured the intellj code config to retain 1 (or was it 2) emply lines in java code fragments - so maybe it's supported for properties too, just wrongly configured. Either way, I suggest reverting property files until we get a decision on that.